libafl_qemu/src/injections.rs

```diff
@@ -0,0 +1,312 @@
+/*
```

Comment:
Usually we have documentation about what a file/module does at the beginning of the file using `//!`

Reply:
ok done